### PR TITLE
Update category/tag flash assers

### PR DESCRIPTION
**Pull request by nzr-zts**
Purpose or Intent
=================
Update category/tag flash assers while crud

{{pytest: -v -k test_tagvis_catalog}}

### CODE REVIEW

File: cfme/configure/configuration/region_settings.py [hunk 1]
```diff
@@ -115,14 +115,11 @@ def create(self, cancel=False):
 
         if cancel:
             view.cancel_button.click()
-            flash_message = 'Add of new Category was cancelled by the user'
```
Comment (quarckster) [inline cfme/configure/configuration/region_settings.py]: **QUESTION**
Why did you remove all flash messages?

Comment (nzr-zts) [reply cfme/configure/configuration/region_settings.py]: As it was discussed before, it's better to check if there is no error than checking the text of the flash message. Flash messages could be checked in a separate test case to decrease the amount of failed test cases by this reason.

Comment (quarckster) [reply cfme/configure/configuration/region_settings.py]: I disagree, @mfalesni , @mshriver, @psav  your opinions.

Comment (mshriver) [reply cfme/configure/configuration/region_settings.py]: Not sure who/where this was discussed, but I would disagree with this opinion and change.

Though changing flash messages cause us some headache and churn for text locators, simply removing these assertions from the framework greatly lowers the quality of existing tests with no replacement for this lost test coverage.

I would consider accepting this if it also included new tests for all of the flash messages, but even then I would be resistant to the change. 

If these flash messages have changed, please submit a PR with version picking for the expected message.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*
